src/data/blob.ts

```diff
@@ -95,13 +94,7 @@ export const loadFromBlob = async (
         elements: clearElementsForExport(data.elements || []),
         appState: {
           theme: localAppState?.theme,
-          fileHandle:
-            blob.handle &&
-            ["application/json", MIME_TYPES.excalidraw].includes(
-              getMimeType(blob),
-            )
-              ? blob.handle
-              : null,
```

Comment:
I've removed this to allow for importing files without an extension. We're doing a check above to see if the file contains an appropriate JSON so we should be fine.